f/simplewoundingsystem/fn_LifeTick.sqf

```diff
@@ -31,22 +31,22 @@ while {alive _unit} do
 		// stops units from not being able to first aid.
 		_unit setdamage 0.26;
 	};
-    if(_downed || _bleeding) then
-    {
-    	// blood loss
-    	_unit setVariable ["f_wound_blood",_blood - 0.6 max 0];
-    	if(damage _unit < 0.251) then { _unit setDamage 0.251};
-    	if(getBleedingRemaining _unit <= 0) then {    	_unit setBleedingRemaining 10;};
-		if(_downed && {animationstate _unit != [_unit] call f_fnc_GetAnimation)} && !{(["heal",animationstate _unit] call bis_fnc_inString)) then 
+  if(_downed || _bleeding) then
+  {
+  	// blood loss
+  	_unit setVariable ["f_wound_blood",(_blood - (0.6 -(random 0.1))) max 0];
+  	if(damage _unit < 0.251) then {_unit setDamage 0.251};
+  	if(getBleedingRemaining _unit <= 0) then {_unit setBleedingRemaining 10;};
+		if(_downed && {(animationstate _unit != [_unit] call f_fnc_GetAnimation)} && !(["heal",animationstate _unit] call bis_fnc_inString)) then
```

Comment:
i see some unnecessary ( brackets ) in the second condition

and some inconsistent use of {lazy} evaluation between second and third condition.